### ISSUE TITLE
Set up releases and allow installation through package managers

### ISSUE THREAD

**Issue by Nevin1901** [tkellogg/dura#32]
Currently there's no easy way to download and install dura. Once the project comes into the later stages, someone should set up GitHub releases, as well as allowing dura to be installed with package managers like brew and apt-get.

**Comment by tkellogg** [comment]
Yes! I'd love to have this. Do I have to do it? Or is that something someone else can contribute?

**Comment by Nevin1901** [comment]
I have no idea how one would go about publishing it to the different package managers, but I can take a look

**Comment by herbygillot** [comment]
I think you can start by making a release available.  If the code is at a some reasonable point, you can create a Github release at the version number that makes the most sense to you, and from that, maintainers will have something they can use to put packages together.

A release not only provides a downloadable asset that a package maintainer can use to create a package, but also a version number for the package. For starters, I can create and maintain the package for `dura` for [MacPorts](https://www.macports.org).

**Comment by drupol** [comment]
The package for Nixos is ready at https://github.com/NixOS/nixpkgs/pull/153803

**Comment by GiantLameChips** [comment]
working on a copr package https://copr.fedorainfracloud.org/coprs/jstyl001/dura/
